tagstudio/src/qt/modals/build_tag.py

```diff
@@ -204,6 +208,16 @@ def set_tag(self, tag: Tag):
 
         self.tag = tag
 
+    def on_name_changed(self):
+        is_empty = not self.name_field.text().strip()
+
+        self.name_field.setStyleSheet(
+            "border: 1px solid red; border-radius: 2px" if is_empty else ""
+        )
```

Comment:
Superficial nitpick: Use the universal "red" UI color defined in `palette.py`
```suggestion
        self.name_field.setStyleSheet(
            f"border: 1px solid {get_ui_color(ColorType.PRIMARY, UiColor.RED)}; border-radius: 2px"
            if is_empty
            else ""
        )
```